include/boost/url/detail/impl/segments_iterator_base.ipp

```diff
@@ -0,0 +1,153 @@
+//
+// Copyright (c) 2019 Vinnie Falco (vinnie.falco@gmail.com)
+// Copyright (c) 2022 Alan de Freitas (alandefreitas@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/CPPAlliance/url
+//
+
+#ifndef BOOST_URL_DETAIL_IMPL_SEGMENTS_ITERATOR_BASE_IPP
+#define BOOST_URL_DETAIL_IMPL_SEGMENTS_ITERATOR_BASE_IPP
```

Comment:
macro name

Reply:
But the path is `impl`. All files in this directory start with `BOOST_URL_DETAIL_IMPL_`.